content/post/2019-03-18-config-multivendor-network-devices.md

```diff
@@ -0,0 +1,52 @@
+---
+date: 2019-03-11T00:00:00-00:00
```

Comment:
```suggestion
date: 2019-03-11T00:00:00-00:00
draft: true
```
Let's mark this as draft so we can get this merged without actually publishing it. When the next release comes out including the openconfig lesson, I'll set a proper date and get this blog out.